### DISCUSSION TITLE
Commande volet roulant fibaro

### DISCUSSION THREAD

**Discussion by lilouane** [zwave-js/zwave-js#3474] [Q&A]
Bonjour,

nouveau sous HA, venant de Jeedom, 
j'ai migrée ma config zwave sous HA, tout fonctionne cependant je remarque que mes commandes de volet roulant sont inversée, et je ne comprend pas pourquoi, je ne trouve rien d'intéressant dans la config non plus
quand mes volet roulant avec mes roller shutel sont ouvert l'icone de HA est fermé et me propose la flèche pour fermer le volet
et inversement quand ils sont fermé l'icone passe a ouvert et me propose l'icone pour fermer le volet ....
une idée ? ps la capture d'ecran jointe montre des volet fermer et demande a les ouvrir ... il sont pourtant bien ouvert !
Par avance merci
![image](https://user-images.githubusercontent.com/91025933/136243797-2751590e-6c71-4b21-8a6d-1c736feb77f8.png)

**Comment by AlCalzone** [comment]
Hi there,

my french isn't that good, so I've translated with https://www.deepl.com:

> Hello,
> 
> new under HA, coming from Jeedom,
> I migrated my zwave configuration to HA, everything works however I notice that my shutter commands are reversed, and I don't understand why, I don't find anything interesting in the configuration either
> when my shutters with my shutel rollers are open the HA icon is closed and offers me the arrow to close the shutter
> and inversely when they are closed the icon changes to open and offers me the icon to close the shutter ....
> any idea? ps the attached screenshot shows closed shutters and asks to open them ... they are however well open!
> Thanks in advance

To answer your question:
Jeedom probably had an option to invert the shutters so the levels 0 and 99 are switched. Z-Wave JS does not do that because it tries to stay as close to the Z-Wave specification as possible. From my experience (I also have Fibaro shutters) this is usually not a problem unless you've wired the shutters incorrectly. If it is not too big of a problem, I suggest you double-check the wiring. Otherwise this would be a feature request for HomeAssistant.

Pour répondre à votre question :
Jeedom avait probablement une option pour inverser les volets afin que les niveaux 0 et 99 soient intervertis. Z-Wave JS ne fait pas cela car il essaie de rester aussi proche que possible de la spécification Z-Wave. D'après mon expérience (j'ai aussi des volets Fibaro), ce n'est généralement pas un problème, sauf si vous avez mal câblé les volets. Si le problème n'est pas trop important, je vous suggère de revérifier le câblage. Sinon, il s'agirait d'une demande de fonctionnalité pour HomeAssistant.
